Modules/OptionHolder.cs

```diff
@@ -892,6 +892,10 @@ public static CustomGameMode CurrentGameMode
     public static OptionItem ImpCanBeEgoist;
     public static OptionItem ImpEgoistVisibalToAllies;
     public static OptionItem EgoistCountAsConverted;
+    public static OptionItem CrewCanBeRainbow;
+    public static OptionItem ImpCanBeRainbow;
+    public static OptionItem NeutralCanBeRainbow;
+    //public static OptionItem RainbowColorChangeTime;
```

Comment:
Same

Reply:
You can move this to Rainbow custom options